data/apply-planet_osm_line.sql

```diff
@@ -33,6 +33,8 @@ CREATE INDEX planet_osm_line_transit_geom_15_index ON planet_osm_line USING gist
 
 CREATE INDEX planet_osm_line_waterway_geom_index ON planet_osm_line USING gist(way) WHERE waterway IN ('canal', 'river', 'stream', 'dam', 'ditch', 'drain');
 
+CREATE INDEX planet_osm_line_piste_geom_index ON planet_osm_line USING gist(way) WHERE tags ? 'piste:type';
```

Comment:
Whoops, I read this too quickly and didn't realize that this already was a partial on the geometry and not just on `piste:type`. I think it's fine the way it is. :+1: